state/workingset.go

```diff
@@ -505,7 +512,7 @@ func (ws *workingSet) getCandidates(height uint64) (CandidateList, error) {
 //======================================
 // private transfer/vote functions
 //======================================
-func (ws *workingSet) handleTsf(tsfs []*action.Transfer) error {
+func (ws *workingSet) handleTsf(producer *Account, tsfs []*action.Transfer, gasLimit *uint64) error {
```

Comment:
If the only use case is to check wether gas is sufficient, I don't think it is necessary to pass in gasLimit here, as the gas sufficiency is already checked in actpool and blockvalidator.